### PR TITLE
Replace next

### PR DESCRIPTION
**Pull request by HenrikJoreteg**
making tweaks per discussion on #64.

This replaces `next` with `jump` which works the same if using defaults, but is more configurable.

### CODE REVIEW

**Comment by HenrikJoreteg** [comment]
ugh, sorry for all the noise in this PR, since it has all the built stuff.

Biggest thing is reviewing doc, implementation and tests for `jump` starting here: https://github.com/AmpersandJS/amp/pull/67/files#diff-c3c914253380116fd0a2e7f0ea642b91R1

Also interested in input on the name and API. <3z

**Comment by kamilogorek** [comment]
Verified https://github.com/AmpersandJS/amp/pull/67/files#diff-c3c914253380116fd0a2e7f0ea642b91R1 + tests and it looks fine! `jump` naming is fine for me as well, as it describes exactly what it does, it jumps in one way or another.

**Comment by aaronmccall** [comment]
+1 on the api, name, and docs

**Comment by fyockm** [comment]
As @kamilogorek mentioned, `jump` makes more sense to me since it doesn't imply direction (vs next/prev). API and docs also look good to me.

File: modules/jump/jump.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+var isNumber = require('amp-is-number');
+var isArray = require('amp-is-array');
+var indexOf = require('amp-index-of');
+
+
+module.exports = function jump(array, currentItem, jumpSize, loop) {
+    if (!isArray(array)) return array;
```
Comment (fyockm) [inline modules/jump/jump.js]: No test covers this line.

Comment (HenrikJoreteg) [inline modules/jump/jump.js]: It's covered here, right? https://github.com/AmpersandJS/amp/pull/67/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR2955

**Comment by HenrikJoreteg** [comment]
Not sure how, but the travis CI hook had gotten disabled somehow. Anyway, re-installed it, seems to be running it.

**Comment by HenrikJoreteg** [comment]
@kamilogorek @aaronmccall @fyockm thanks everyone! Merged, published, and site-updated.

File: modules/jump/jump.js (hunk 1 above)
Comment (fyockm) [inline modules/jump/jump.js]: Yes, sorry. I must have been looking at the `example` file instead of the `test` file. My apologies.

**Pull request merged**